R/no_tab_linter.R

```diff
@@ -18,6 +21,10 @@
 #'
 #' @evalRd rd_tags("no_tab_linter")
 #' @seealso [linters] for a complete list of linters available in lintr.
+#'
+#' @references
+#' - https://www.jwz.org/doc/tabs-vs-spaces.html
```

Comment:
my only concern is that blogs like this tend to wind up as a maintenance headache in the long term as redirects evolve / pages die.

but fine to leave it in for now